Tensile/Source/TensileTypes.h

```diff
@@ -225,7 +225,7 @@ class ProblemDims
 
     // Constructor accepts variable number of sizes:
     template <typename... Ts>
-    ProblemDims(Ts... args)
```

Comment:
cppcheck: Class 'ProblemDims' has a constructor with 1 argument that is not explicit.

Reply:
I might delete this and the other "old client" files in a separate PR